spec/spec_helper.rb

```diff
@@ -27,8 +27,21 @@
 require 'action_controller/railtie'
 require 'rspec/rails'
 
+module MultiTenantTest
+  class Application < Rails::Application; end
+end
+
+# NOTE: Specifies columns which shouldn't be exposed while calling #inspect.
+ActiveSupport.on_load(:active_record) do
+  self.filter_attributes += MultiTenantTest::Application.config.filter_parameters
+end
+
 require 'activerecord_multi_tenant'
 
+MultiTenantTest::Application.config.secret_token = 'x' * 40
+MultiTenantTest::Application.config.secret_key_base = 'y' * 40
+MultiTenantTest::Application.config.filter_parameters = [:password]
```

Comment:
It called before `ActiveRecord::Base`. So, I moved to here.

Reply:
Could you add a comment explaining this necessity? Could you refer the test in the comment as well?

Reply:
Ok, I commented.